core/src/test/java/io/knotx/server/handler/http/response/writer/ServerResponseTest.java

```diff
@@ -41,6 +41,8 @@
 class ServerResponseTest {
 
   private static final Set<String> DEFAULT_ALLOWED_HEADERS = Collections.singleton("test");
+  private static final Set<String> WILDCARD_ALLOWED_HEADERS = Collections.singleton("*");
```

Comment:
Actually I'm not 100% sure if this change requires upgrade notes?
(Was `*` supported before, or `.*` was the correct one?)

Reply:
```*``` Wasn't supported, and now also shouldn't be. 

We should use ```.*```

I fixed this in tests